### PR TITLE
Add Subscription approval capability to Publisher portal

### PR DESCRIPTION
**Pull request by shilmyhasan**
Add Subscription approval capability to Publisher portal.

image

image

### CODE REVIEW

File: portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/ContentBase.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (lasanthaS) [inline portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/ContentBase.jsx:4]: Change this to WSO2 LLC

Comment (lasanthaS) [reply portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/ContentBase.jsx:4]: Check other places as well

File: portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/InputListBase.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react';
```
Comment (lasanthaS) [inline portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/InputListBase.jsx:1]: Add license header

**Review by lasanthaS — CHANGES_REQUESTED** [review]

File: portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/ContentBase.jsx (hunk 1 above)
Comment (shilmyhasan) [reply portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/ContentBase.jsx]: Refactored

File: portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/InputListBase.jsx (hunk 2 above)
Comment (shilmyhasan) [reply portals/publisher/src/main/webapp/source/src/app/components/Addons/Addons/InputListBase.jsx:19]: Refactored

**Review by lasanthaS — APPROVED** [review]

**Pull request merged**